app/src/main/res/values/strings.xml

```diff
@@ -569,6 +569,7 @@
 
     <!-- strings related to share images -->
     <string name="sorry_msg">Pictures without barcode cannot yet be added with the app. Please try the mobile web version.</string>
+    <string name="type_the_barcode_here_label" translatable="false">Type the barcode here</string>
```

Comment:
```suggestion
    <string name="type_the_barcode_here_label">Type the barcode here</string>
```

Reply:
@VaiTon Is there any reason of removing `translatable="false"`. , When do we need to make it `true` and `false` ?

Reply:
After doing some exploration , I found that , when we use `translatable="false"`, it means that it will not be translated irrespective of the language (in case of multiple languages), so it will remain same in all states.

Reply:
Yeah, and that's not what we want in this case